Unix/pal/palcommon.h

```diff
@@ -22,7 +22,7 @@
 #    include <windows.h>
 #    include <strsafe.h>
 #else
-#    include "../../common/linux/sal.h"
+#    include "linux/sal.h"
```

Comment:
Originally header files within <> are supposed to be system headers, and those in quotes "" are supposed to be local ones, but since compiler doesn't enforce this, people have used them interchangeably.  In OMI, I've seen both <> and "" local header files.

Technically it's "common/Linux/sal.h", but both works.